### ISSUE TITLE
"The input element's type ('checkbox') does not support selection."

### ISSUE THREAD

**Issue by craigdallimore** [azaslavsky/domJSON#18]
Hi! Great library.

Not sure if this is a bug or a browser issue, but I think you might be interested - I seem to be having issues JSONifying a few types of input elements. [Here is a JSFiddle](https://jsfiddle.net/decoy9697/9fLgxqch/) demonstrating what can happen.

I'm using chrome n 43.0.2357.124 (64-bit).

It's as if `checkbox` (and `radio`, `file`, etc) will raise an exception if you so much as _access_ `selectionStart` like so:

```
'selectionStart' in node // true
node['selectionStart'] // DOMException
```

... which is more or less what I think is happening [here](https://github.com/azaslavsky/domJSON/blob/master/src/domJSON.js#L424).

**Comment by azaslavsky** [comment]
Huh, interesting.... i'll have to take a closer look.  Thanks for the heads up!

**Comment by azaslavsky** [comment]
This issue has been resolved by commit #32
